tests/unit/clients/test_gdns.py

```diff
@@ -86,26 +86,31 @@ def test_get_rrsets_as_objects(rrset_dict):
 
 
 @pytest.mark.asyncio
+@pytest.mark.parametrize('params', [
+    None,
+    {'fields': 'some-other-field,another-field'},
+    {'name': 'example.com.', 'type': 'A'}
+])
 async def test_get_records_for_zone(fake_response_data, client, caplog,
-                                    monkeypatch):
-    mock_get_json_called = 0
+                                    mocker, params):
```

Comment:
mocker doesn't seem to be used

Reply:
Hmm, nope, wonder what was going on there...  Good catch!